openmc/data/neutron.py

```diff
@@ -824,7 +824,7 @@ def from_endf(cls, ev_or_filename):
         return data
 
     @classmethod
-    def from_njoy(cls, filename, temperatures=None, **kwargs):
+    def from_njoy(cls, filename, temperatures=None, error=0.001, **kwargs):
```

Comment:
Let's put this argument on `njoy.make_ace` instead of here. It will still be possible for a user to specify it as a keyword argument since `**kwargs` gets passed to `make_ace`.